src/prefect/core/flow.py

```diff
@@ -145,7 +145,8 @@ def __init__(
         state_handlers: List[Callable] = None,
         on_failure: Callable = None,
         validate: bool = None,
-        result_handler: ResultHandler = None,
+        result_handler: Optional[ResultHandler] = None,
+        result: Optional[Result] = None,
```

Comment:
Is this `result` being used anywhere?

Reply:
I'm for changing the deprecation warning to specify that `result_handler` will be deprecated in 0.11.0 in favor of `result` and no longer a valid kwarg in 0.12.0 (we can discuss tomorrow if those are really the true release dates, but its the current proposal). In addition we could switch it out to this stronger language in 0.11.0.

Reply:
Agreed, we'll finalize this copy after release planning